projects/public-app-getting-started-template/src/app/extensions/ExampleCard.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { List, Text, Link, Alert } from "@hubspot/ui-extensions";
+import { hubspot } from "@hubspot/ui-extensions";
+
+hubspot.extend(() => <Extension />);
+
+const Extension = () => {
+  return (
+    <>
+      <Text>
+        Congrats! You just deployed your first App card. What's next? Here are
+        some pointers to get you started:
+      </Text>
+      <List variant="unordered-styled">
+        <Link href="https://developers.hubspot.com/docs/platform/ui-components">
+          📖 Explore our library of UI components
+        </Link>
+        <Link href="www.developers.hubspot.com">
+          📖 Learn more about utilities to help you build better extensions
+        </Link>
+        <Link href="github.com/hubspot/ui-extensions-examples">
+          📖 Use existing private app code samples to get inspiration
+        </Link>
+        <Link href="https://www.youtube.com/watch?v=FDumsLFwyNM&list=PLTGNq2fWP3b0HpN11u4CuM6tNJBacORWs">
+          {" "}
+          ▶️ Watch existing video tutorials to use CRM development tools to
+          build UI extensions (Private apps){" "}
```

Comment:
We specify private apps here, should we change this?

Reply:
The youtube video series linked here is about private apps (which I'm assuming we're linking to because we don't have videos on public apps yet). Going to resolve this comment.